src/TabsPage.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage = ({ match }) => {
+  const tabsId = match.params.tabsId;
```

Comment:
use useParams hook for getting tabId

https://reactrouter.com/web/api/Hooks/useparams